src/mscorlib/System.Private.CoreLib.csproj

```diff
@@ -276,7 +275,6 @@
     <Compile Include="$(BclSourcesRoot)\System\Runtime\InteropServices\WindowsRuntime\CLRIKeyValuePairImpl.cs" />
   </ItemGroup>
   <ItemGroup>
-    <Compile Include="$(BclSourcesRoot)\System\AggregateException.cs" />
```

Comment:
I think we forgot to add blobs for exception types. I recall that in an offline thread we also got info from the ILC team that the serialized StackTrace might not be exchangeable between ILC and Core and Desktop. We at least should at test blobs for the two Exception types.